rollup.config.js

```diff
@@ -1,104 +1,131 @@
-import autoprefixer from 'autoprefixer';
-import path from 'path';
-import postcssImport from 'postcss-import';
-import rimraf from 'rimraf';
-import alias from '@rollup/plugin-alias';
 import commonjs from '@rollup/plugin-commonjs';
 import json from '@rollup/plugin-json';
 import postcss from 'rollup-plugin-postcss';
+import postcssImport from 'postcss-import';
+import tailwindcss from 'tailwindcss';
+import autoprefixer from 'autoprefixer';
 import resolve from '@rollup/plugin-node-resolve';
-import svelte from 'rollup-plugin-svelte';
-import terser from '@rollup/plugin-terser';
 import typescript from '@rollup/plugin-typescript';
-import preprocess from 'svelte-preprocess';
-import tailwindcss from 'tailwindcss';
+import path from 'path';
+import svelte from 'rollup-plugin-svelte';
+import alias from '@rollup/plugin-alias';
+import sveltePreprocess from 'svelte-preprocess';
 import aliases from './alias.config.js';
 
 const production = !process.env.ROLLUP_WATCH;
+const packageFolder = 'package';
+
+const plugins = [
+  alias({
+    entries: aliases,
+  }),
+  postcss({
+    extract: path.resolve('./package/widget.css'),
+    plugins: [postcssImport, tailwindcss, autoprefixer],
+  }),
+  svelte({
+    compilerOptions: {
+      // enable run-time checks when not in production
+      dev: !production,
+    },
 
-export default {
-  input: ['src/widget/modal.svelte', 'src/widget/inline.svelte'],
-  onwarn: (warning) => {
-    // TODO: Revisit once the JS SDK is updated to pure ES Modules
-    if (warning.code === 'THIS_IS_UNDEFINED') {
-      return;
-    }
+    emitCss: false,
```

Comment:
May I ask why is this `false`?

Reply:
Was this property supposed to be added to the `rollup.types.config.js`?

Reply:
was mostl yme trying different things. It probably should be `true` since we use the postcss plugin.